### PR TITLE
Adding Configuring template repos

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Update for https://github.com/eclipse/codewind/issues/32 and https://github.com/eclipse/codewind/issues/434.

There's not much to this PR right now, but hopefully we can add more to this page if/when the UI evolves. Feel free to suggest any other additions if you have them. Thanks!

### CODE REVIEW

**Comment by tetchel** [comment]
Well, I think we're more interested in explaining here what "template repositories" actually are?

File: docs/_documentations/mdt-vsc-usingadifferenttemplate.md [hunk 1]
```diff
@@ -22,3 +22,12 @@ When you create a new project, you can choose from the default set of templates
 ```
 
 After you have edited and saved this file, restart Codewind.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: The above info is outdated and should be removed, the new UI replaces editing that file.

**Comment by tetchel** [comment]
also, eclipse has the same feature.

**Comment by sishida** [comment]
@tetchel 
- Added sentence about template repositories.
- Removed outdated line 24.
- Updated corresponding Eclipse file.

**Comment by tetchel** [comment]
Everything above line 24 is outdated not just that particular line. Except for

> When you create a new project, you can choose from the default set of templates available, or you can choose from a template repository of your own.

which is still accurate

**Comment by sishida** [comment]
@tetchel and @eharris369 I have removed the outdated info. Sorry about my misunderstanding! 😅

**Comment by tetchel** [comment]
One other correction is that we have moved from calling them "template repositories" to "template sources". Similarly, the action to open the Manage page is called `Manage Template Sources` in VS Code.

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 2]
```diff
@@ -10,15 +10,10 @@ order: 8
...
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
+- Click **Add New** to add a repository source to the table.
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: Click **Manage Repositories** ( I probably need to change this to Manage Template Sources in the next release)

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 3]
```diff
@@ -10,15 +10,10 @@ order: 8
...
+- Enable or disable a repository type to view the repositories of that type in the wizard.
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: In the dialog, use the check boxes to enable or disable repositories.  Use the **Add** button to a new repository.  The **Remove** button can be used to remove non-default repositories.

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 4]
```diff
@@ -10,15 +10,10 @@ order: 8
...
 
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: Maybe call it Repository Manager for now until I change it to Template Source Manager.

Comment (eharris369) [reply docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: Ignore this as I am changing the strings right now.

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 5]
```diff
@@ -10,15 +10,10 @@ order: 8
 parent: usingcodewind
 ---
 
-# Using a template repository of your own
+# Configuring template repositories
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: Should be Configuring template sources

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: Using a template repository of your own
-description: Using a template repository of your own
+title: Configuring template repositories
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: Configuring template sources (other places too)

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md (hunk 5 above)
Comment (sishida) [reply docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: It should be displaying as such in the latest commit. :)

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md (hunk 6 above)
Comment (sishida) [reply docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: It should be displaying as such in the latest commit. :)

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 7]
```diff
@@ -1,24 +1,21 @@
...
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Repository Manager, right-click **Projects (Local)**.
+- Click **Manage Repositories** to add a source to the table.
+- In the dialog window, use the check boxes to enable or disable repositories:
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: The checkboxes are separate from Add and Remove.  Rewrite as:
In the dialog window:
- Use the check boxes to enable or disable template sources
- Use Add to add a new template source
- Use Remove to remove non-default template sources

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 8]
```diff
@@ -1,24 +1,21 @@
...
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Repository Manager, right-click **Projects (Local)**.
+- Click **Manage Repositories** to add a source to the table.
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: Change to **Mange Template Sources**

**Comment by sishida** [comment]
@tetchel and @eharris369 Hi, team,

Erin, I have addressed your latest comments in my new commit.

Also, I have asked Guy for review. Let's go ahead and merge if the content looks OK and then I'll create a new PR if Guy asks for any changes.

File: docs/_documentations/mdt-eclipse-usingadifferenttemplate.md [hunk 9]
```diff
@@ -1,24 +1,22 @@
...
 
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Repository Manager, right-click **Projects (Local)**.
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-usingadifferenttemplate.md]: To open the **Template Source Manager**

**Comment by sishida** [comment]
@eharris369 Latest comment has been addressed! :)

File: docs/_documentations/mdt-vsc-usingadifferenttemplate.md [hunk 10]
```diff
@@ -1,24 +1,19 @@
...
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
+- Click **Manage Template Sources** to add a source to the table.
+- Enable or disable a source type to view the sources of that type in the wizard.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: What does this line mean?

Comment (tetchel) [reply docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: maybe "Enable or disable sources to set if that source's templates should appear in the Create Project wizard"

Comment (sishida) [reply docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: Yes, Guy mentioned a wizard. Maybe something like...
```
- Enable a source to make that source template appear in the Create Project wizard.
- Disable a source to remove that source template from the Create Project wizard.
```

Comment (tetchel) [reply docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: yes, although i'm not sure something that simple requires so many words to explain :)

**Comment by sishida** [comment]
@tetchel I have added a new commit! :)

File: docs/_documentations/mdt-vsc-usingadifferenttemplate.md [hunk 11]
```diff
@@ -1,24 +1,20 @@
...
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
+- Click **Manage Template Sources** to add a source to the table.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: sorry, i missed this one last time.

> - Click **Manage Template Sources** to add a source to the table.

It should be something like.
Click Manage Template Sources to open the Template Sources wizard.
Click Add New to add a source to the table.

Comment (sishida) [reply docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: Added new commit!  👍

Comment (sishida) [reply docs/_documentations/mdt-vsc-usingadifferenttemplate.md]: Added a new commit! 👍

**Comment by sishida** [comment]
@tetchel Good idea. I have opened https://github.com/eclipse/codewind/issues/497.

**Comment by AndyRWatson** [comment]
we also need to document how you create a template in your own template repository https://github.com/eclipse/codewind/issues/713

**Pull request merged**